frontend/src/App.res

```diff
@@ -18,19 +18,44 @@ query ($startDate: timestamp!, $endDate: timestamp!) {
   }
 `)
 
-let comparePulls = ((pn1, _b1), (pn2, _b2)) => {
-  -compare(pn1, pn2)
+module PullCompare = Belt.Id.MakeComparable({
+  type t = (int, option<string>)
+  let cmp = compare
+})
+
+let collectBenchmarksForRepo = (~repo, data: array<GetBenchmarks.t_benchmarks>): array<
+  GetBenchmarks.t_benchmarks,
+> => {
+  data->Belt.Array.keep(item => item.repo_id == repo)
 }
 
-let collectPulls = (data: array<GetBenchmarks.t_benchmarks>): array<(int, option<string>)> => {
-  let data = Belt.List.fromArray(data)
-  let data = Belt.List.keepMap(data, (item: GetBenchmarks.t_benchmarks) =>
+let collectPullsForRepo = (~repo, benchmarks: array<GetBenchmarks.t_benchmarks>): array<(
```

Comment:
I think this is ok for now while we have two repos. Once we have more we should pass the `repo_id` to the GraphQL query.

Happy to merge this as is, just something to keep in mind for the follow up work.